src/app/task/task.component.html

```diff
@@ -1,25 +1,25 @@
-<h2>Task To Do: </h2>
+<h2 class="task-to-do-title">Task To Do: </h2>
```

Comment:
Same thing here. If you are adding element specific styles, add id to it and add styles using this id.